mesh_sequence_controller.py

```diff
@@ -617,4 +637,4 @@ def unregister():
     bpy.types.INFO_MT_mesh_add.remove(menu_func)
 
 if __name__ == "__main__":
-    register()
\ No newline at end of file
+    register()

```

Comment:
What changed on this line? I can't tell from the Github diff view.

Reply:
Your editor does't add a '\n' at the end of the last line, while my vim editor added.

Reply:
The difference is whether there is a '\n' character at the end of last line. Different editors works different. Vim seems to automatically add such a character.